src/app/map/[id]/components/TurfVisibilityManager.tsx

```diff
@@ -0,0 +1,87 @@
+import { useContext, useMemo } from "react";
+import { Layer, Source } from "react-map-gl/mapbox";
+import { MarkerAndTurfContext } from "@/app/map/[id]/context/MarkerAndTurfContext";
+import { useMapViews } from "@/app/map/[id]/hooks/useMapViews";
+import { mapColors } from "../styles";
+import type { FeatureCollection, Polygon } from "geojson";
+
+export default function TurfVisibilityManager() {
```

Comment:
question: I thought that the turfs were moved out of a <Layer> and into the Mapbox draw so that they can be edited. Does editing a turf still work?